src/react/components/pages/editorPage/assetPreview.tsx

```diff
@@ -45,11 +33,11 @@ export default class AssetPreview extends React.Component<IAssetPreviewProps, IA
                     </div>
                 }
                 {asset.type === AssetType.Image &&
-                    <img src={asset.path} onLoad={this.onAssetLoad} />
+                    <img src={`file://#${asset.path.replace(/(^\w+:|^)\/\//, "")}`} onLoad={this.onAssetLoad} />
```

Comment:
This is always prefixing asset path with the file:// protocol but if the sourceInput is configured with something like bingImageSearch the asset path strings should already have the http:// or https:// protocol string and so we should not replace this with file://